### PR TITLE
feat: Allow specifying project open behaviour after creating new project

### LINKED ISSUE #1278: 这个插件能不能像Maven for Java插件一样提供一个创建项目后的行为选项,Maven: Project Open Behavio 这个选项.
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by CsCherrYY**
fix #1278, #1166

### CODE REVIEW

File: extension/src/commands/CreateProjectCommand.ts [hunk 1]
```diff
@@ -48,12 +49,38 @@ export class CreateProjectCommand extends Command {
...
+                    }
+                }
+                if (openProjectBehaviour === ProjectOpenBehaviourValue.OPEN) {
+                    vscode.commands.executeCommand(
+                        "vscode.openFolder",
+                        vscode.Uri.file(metadata.targetFolder),
+                        hasOpenFolder
+                    );
+                } else if (openProjectBehaviour === ProjectOpenBehaviourValue.ADDTOWORKSPACE) {
+                    vscode.workspace.updateWorkspaceFolders(folders!.length, null, {
```
Comment (Eskibear) [inline extension/src/commands/CreateProjectCommand.ts]: what if here `folders` is undefined. Say the setting is pointed to "ADDTOWORKSPACE", and you create a project without opening any folder.

Comment (CsCherrYY) [reply extension/src/commands/CreateProjectCommand.ts]: See: https://github.com/microsoft/vscode-gradle/pull/1282/files#diff-99dbae36046fe7b4147fc646659041025bc994a4b7762f5e08713881e87e0cebR61 we will not show this option when `folders` is undefined.

Comment (Eskibear) [reply extension/src/commands/CreateProjectCommand.ts]: https://github.com/microsoft/vscode-gradle/pull/1282/files#diff-99dbae36046fe7b4147fc646659041025bc994a4b7762f5e08713881e87e0cebR54
You won't enter that block if the setting is explicitly set to `ADDTOWORKSPACE`

Comment (CsCherrYY) [reply extension/src/commands/CreateProjectCommand.ts]: you're right, let me correct it.

**Review by Eskibear — APPROVED** [review]

**Pull request merged**